doc/manual.rst

```diff
@@ -5453,6 +5468,145 @@ powerful programming construct that still suffices. So the "check list" is:
 (4) Else: Use a macro.
 
 
+Alias Declaration
+=================
```

Comment:
There is considerable overlap with a nullary, non-overloaded template. More on the RFC.

Reply:
I don't see how conflating with nullary templates would work, let alone improve, but marking this comment as resolved since this is discussed here: https://github.com/nim-lang/RFCs/issues/232#issuecomment-654000250